docs/website-launch-buttons.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Launch Jupyter sessions
+---
+
+If your source file is a Jupyter Notebook (`.ipynb`), you can add a button to your MyST website that launches a live Jupyter server in the cloud, and opens that content so they can edit and interact with it.[^thebe]
```

Comment:
```suggestion
You can add a button to your MyST website that launches a Jupyter environment in the cloud and opens the current file in the editor. If the file is a Jupyter Notebook (`.ipynb` file) or a MyST document with code-cells[^lab-myst], users can use the Jupyter environment to run the code cells and inline expressions.[^thebe]
```

Reply:
The important points I'm trying to make here are:

1. You don't actually need the current doc to be executable (my mistake over Slack) for the button to appear on a page.
2. If the environment understands MyST documents with kernelspecs, they too can be executed.